src/InstrumentationEngine/Instruction.cpp

```diff
@@ -362,221 +356,144 @@ HRESULT MicrosoftInstrumentationEngine::CInstruction::SetIsRemoved()
 {
     HRESULT hr = S_OK;
-    CLogging::LogMessage(_T("Starting CInstruction::GetAlternateOrdinalOpcode"));
     IfNullRetPointer(pFlags);
 
     *pFlags = s_ilOpcodeInfo[m_opcode].m_flags;
-
-    CLogging::LogMessage(_T("End CInstruction::GetAlternateOrdinalOpcode"));
-
     return S_OK;
 }
 
 HRESULT MicrosoftInstrumentationEngine::CInstruction::GetInstructionLength(_Out_ DWORD* pdwLength)
 {
     HRESULT hr = S_OK;
-    CLogging::LogMessage(_T("Starting CInstruction::GetInstructionLength "));
     IfNullRetPointer(pdwLength);
 
     *pdwLength = s_ilOpcodeInfo[m_opcode].m_opcodeLength + s_ilOpcodeInfo[m_opcode].m_operandLength;
-
-    CLogging::LogMessage(_T("End CInstruction::GetInstructionLength "));
-
     return S_OK;
 }
 
 HRESULT MicrosoftInstrumentationEngine::CInstruction::GetOpcodeLength(_Out_ DWORD* pdwLength)
 {
     HRESULT hr = S_OK;
-    CLogging::LogMessage(_T("Starting CInstruction::GetOpcodeLength"));
     IfNullRetPointer(pdwLength);
 
     *pdwLength = s_ilOpcodeInfo[m_opcode].m_opcodeLength;
-
-    CLogging::LogMessage(_T("End CInstruction::GetOpcodeLength"));
-
     return S_OK;
 }
 
 HRESULT MicrosoftInstrumentationEngine::CInstruction::GetOperandType(_Out_ ILOperandType* pType)
 {
     HRESULT hr = S_OK;
-    CLogging::LogMessage(_T("Starting CInstruction::GetOpcodeLength"));
     IfNullRetPointer(pType);
 
     *pType = s_ilOpcodeInfo[m_opcode].m_type;
-
-    CLogging::LogMessage(_T("End CInstruction::GetOpcodeLength"));
-
     return S_OK;
 }
 
 HRESULT MicrosoftInstrumentationEngine::CInstruction::GetOperandLength(_Out_ DWORD* pdwLength)
 {
     HRESULT hr = S_OK;
-    CLogging::LogMessage(_T("Starting CInstruction::GetOpcodeLength"));
     IfNullRetPointer(pdwLength);
 
     *pdwLength = s_ilOpcodeInfo[m_opcode].m_operandLength;
-
-    CLogging::LogMessage(_T("End CInstruction::GetOpcodeLength"));
-
     return S_OK;
 }
 
 // set to true if this instruction did not exist in the original instruction stream.
 HRESULT MicrosoftInstrumentationEngine::CInstruction::GetIsNew(_Out_ BOOL* pbValue)
 {
     HRESULT hr = S_OK;
-    CLogging::LogMessage(_T("Starting CInstruction::GetIsNew"));
     IfNullRetPointer(pbValue);
 
     *pbValue = m_instructionGeneration == InstructionGeneration::Generation_New ? TRUE : FALSE;
-
-    CLogging::LogMessage(_T("End CInstruction::GetIsNew"));
-
     return S_OK;
 }
 
 // set to true of this instruction has been replaced in the instruction stream.
 HRESULT MicrosoftInstrumentationEngine::CInstruction::GetIsRemoved(_Out_ BOOL* pbValue)
 {
     HRESULT hr = S_OK;
-    CLogging::LogMessage(_T("Starting CInstruction::GetIsRemoved"));
     IfNullRetPointer(pbValue);
 
     *pbValue = m_bIsRemoved;
-
-    CLogging::LogMessage(_T("End CInstruction::GetIsRemoved"));
-
     return S_OK;
 }
 
-HRESULT MicrosoftInstrumentationEngine::CInstruction::GetInstructionSize(_Out_ DWORD* pdwSize)
+DWORD MicrosoftInstrumentationEngine::CInstruction::GetInstructionSize()
 {
-    HRESULT hr = S_OK;
```

Comment:
Maybe add a comment to indicate that GetInstructionSize is no longer implemented in terms of GetOpcodeLength/GetOperandLength